oap-server/server-configuration/configuration-nacos/src/main/java/org/apache/skywalking/oap/server/configuration/nacos/NacosConfigWatcherRegister.java

```diff
@@ -92,8 +91,35 @@ public NacosConfigWatcherRegister(NacosServerSettings settings) throws NacosExce
 
     @Override
     public Optional<GroupConfigTable> readGroupConfig(final Set<String> keys) {
-        // TODO: implement readGroupConfig
-        return Optional.empty();
+        Set<String> keys1 = new HashSet<>();
```

Comment:
```suggestion
        Set<String> keys = new HashSet<>();
```

Is `s` suffix a typo?